### DISCUSSION TITLE
How to get houston in my astro project?

### DISCUSSION THREAD

**Discussion by yuxxeun** [withastro/docs#3025] [Ideas]
Previously, when im using astro version `1.x` there's no houston in there, and now, i was migrating thats project to astro version `2.x` but still not there houston.

**Comment by delucis** [comment]
Hey @yuxxeun! Do you mean on the command line when running `create astro`?

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

Due to the inactivity since Chris question, I think we can safely close this one!

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
